dashboard/test/ui/features/teacher_tools/level_types/multiple_choice_contained_levels.feature

```diff
@@ -66,16 +66,16 @@ Scenario: Teacher can reset progress on multiple choice contained level
   And I am on "http://studio.code.org/s/allthethings/lessons/41/levels/2"
   And I wait for the lab page to fully load
   Then I press "unchecked_0"
-  And I wait up to 5 seconds for element "#checked_0" to be visible
+  And I wait until element "#checked_0" is visible
```

Comment:
Does this always fail on the first one or are there times where it will time out on the later calls? If it always fails on the first one, we could remove some latency by only waiting the full amount on the first one.

Reply:
Good catch, I think it makes sense to keep the limit when selecting the solution as that doesn't involve any networking call.